packages/account/src/Sections/Security/ClosingAccount/closing-account-pending-conditions/closing-account-pending-positions.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { CFD_PLATFORMS, getCFDAccount, getCFDAccountDisplay, getCFDPlatformLabel } from '@deriv/shared';
+import { observer, useStore } from '@deriv/stores';
+import { Localize } from '@deriv/translations';
+import { TCFDPlatform, TDetailsOfDerivXAccount, TDetailsOfMT5Account } from 'Types';
+import ClosingAccountPendingWrapper from './closing-account-pending-wrapper';
+import ClosingAccountPendingContent from './closing-account-pending-content';
+
+type TClosingAccountPendingPositionsProps = {
+    platform: TCFDPlatform;
+    open_positions: TDetailsOfMT5Account[] | TDetailsOfDerivXAccount[];
```

Comment:
**NIT**: Type can be written as `Array<TDetailsOfMT5Account | TDetailsOfDerivXAccount>`